src/physics/cam/cam_diagnostics.F90

```diff
@@ -2021,6 +2044,37 @@ subroutine diag_physvar_ic (lchnk,  pbuf, cam_out, cam_in)
 
       end if
 
+
+      !GC
+      if (ts_idx > 0) then
+        call pbuf_get_field(pbuf, ts_idx, conv_var_2d ,(/1,itim_old/), (/pcols,1/))
+        call outfld('TS&IC   ',conv_var_2d, pcols,lchnk)
```

Comment:
Is there an `addfld` that matches this `outfld`? I have the same question about the four other `outfld` calls below.

Reply:
My point is that you are missing an `addfld` call for these fields.